config.schema.json

```diff
@@ -9,6 +9,22 @@
       "description": "If 'true', ctrl+click on a field to edit.",
       "default": false
     },
+    "min-width": {
+      "type": "integer",
+      "title": "Minimal width",
+      "default": 10
+    },
+    "min-height": {
+      "type": "integer",
+      "title": "Minimal height",
+      "default": 10
+    },
+    "resizable-style": {
+      "type": "string",
+      "title": "Resizable style",
+      "description": "CSS style applied to images when they are resizable",
+      "default": "border:1px dashed black;"
+    },
```

Comment:
What about "preserve_ratio"? 

You'll also need to add these to config.json itself.

Reply:
thanks. It seems I did simply forgot them